en/backup-to-gcs-using-br.md

```diff
@@ -0,0 +1,287 @@
+
+    - `spec.br.cluster`: The name of the cluster to be backed up.
+    - `spec.br.clusterNamespace`: The `namespace` of the cluster to be backed up.
+    - `spec.br.logLevel`: The log level (`info` by default).
+    - `spec.br.statusAddr`: The listening address through which BR provides statistics. If not specified, BR does not listen on any status address by default.
+    - `spec.br.concurrency`: The number of threads used by each TiKV process during backup. Defaults to `4` for backup and `128` for restore.
+    - `spec.br.rateLimit`: The speed limit, in MB/s. If set to `4`, the speed limit is 4 MB/s. The speed limit is not set by default.
+    - `spec.br.checksum`: Whether to verify the files after the backup is completed. Defaults to `true`.
+    - `spec.br.sendCredToTikv`: Whether the BR process passes its GCP privileges to the TiKV process. Defaults to `true`.
+    </details>
+
+    This example backs up all data in the TiDB cluster to GCS. Some parameters in `spec.gcs` can be ignored, such as `location`, `objectAcl`, and `storageClass`.
+
+    The `projectId` in the configuration is the unique identifier of a user project on GCP. For how to obtain the identifier, see [GCP Documentation](https://cloud.google.com/resource-manager/docs/creating-managing-projects).
+
+    <details>
+    <summary>Configure <code>storageClass</code>.</summary>
+
+    GCS supports the following `storageClass` types:
+
+    * `MULTI_REGIONAL`
+    * `REGIONAL`
+    * `NEARLINE`
+    * `COLDLINE`
+    * `DURABLE_REDUCED_AVAILABILITY`
+
+    If you do not configure `storageClass`, the default type is `COLDLINE`. See [GCS Documentation](https://cloud.google.com/storage/docs/storage-classes) for details.
+
+    </details>
+
+    <details>
+    <summary>Configure the object access-control list (ACL) strategy</summary>
+
+    GCS supports the following ACL strategies:
+
+    * `authenticatedRead`
+    * `bucketOwnerFullControl`
+    * `bucketOwnerRead`
+    * `private`
+    * `projectPrivate`
+    * `publicRead`
+
+    If you do not configure the object ACL strategy, the default strategy is `private`. See [GCS Documentation](https://cloud.google.com/storage/docs/access-control/lists) for details.
+    </details>
+
+2. After creating the `Backup` CR, use the following command to check the backup status:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl get bk -n test1 -owide
+    ```
+
+<details>
+
+<summary>More descriptions of fields in the <code>Backup</code> CR</summary>
+
+* `.spec.metadata.namespace`: The namespace where the `Backup` CR is located.
+* `.spec.tikvGCLifeTime`: The temporary `tikv_gc_lifetime` time setting during the backup. Defaults to 72h.
+
+    Before the backup begins, if the `tikv_gc_lifetime` in the TiDB cluster is smaller than the `spec.tikvGCLifeTime` set by the user, TiDB Operator adjusts `tikv_gc_lifetime` to `spec.tikvGCLifeTime`. This operation makes sure that the backup data is not garbage collected by TiKV.
```

Comment:
The related link (dumpling-overview.md) is not available in English yet.
I'll add those link when this PR (https://github.com/pingcap/docs-cn/pull/3800) is translated.

Reply:
I think we can copy the command here, and no need to link to that doc.